SConscript

```diff
@@ -168,7 +168,8 @@ if not env_etc.no_boost_python and hasattr(env_etc, "boost_adaptbx_include"):
         LIBS=env_etc.libs_python
         + env_etc.libm
         + env_etc.dxtbx_libs
-        + env_etc.dxtbx_hdf5_libs,
+        + env_etc.dxtbx_hdf5_libs
```

Comment:
missing comma?

Reply:
Oh, maybe we don't need commas in `SConscript`?

Reply:
I assume we are concatenating a set of lists here?

Reply:
The comma was here from black because multiline function calls always have a trailing comma added, so that adding new parameters minimizes diff. In this case, it's adding a line to the concatenation within a single function call, so the comma moves down